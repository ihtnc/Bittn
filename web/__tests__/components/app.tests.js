import React from 'react';
import renderer from 'react-test-renderer';

import App from "@components/app"

const shallow = (comp) => renderer.create(comp).toJSON();

describe("App component", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot();
  });
});