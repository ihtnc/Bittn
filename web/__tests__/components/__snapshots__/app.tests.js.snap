// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should match latest snapshot 1`] = `
<section
  className="css-xjel9q-AppWrapper e1q1sy7m0"
>
  <div
    className="css-smrfxc-Wrapper e1ehg2020"
  >
    <div
      className="css-1yo68oz-Header e1ehg2024"
    >
      You think you've been bitten?
    </div>
    <div
      className="css-74rito-Label e1ehg2028"
    >
      What happened?
    </div>
    <div
      className="css-12lwwuy-FieldWrapper e1ehg2021"
    >
      <input
        className="css-51f26o-Input e1ehg20211"
        onChange={[Function]}
        value=""
      />
      <span
        className="css-1kefsjr-ExpandableButtonWrapper e1ehg2023"
        disabled={false}
      >
        <span
          className="css-dlo9cb-Button e1ehg20213"
          disabled={true}
          onClick={[Function]}
          title="Next"
        />
      </span>
    </div>
  </div>
</section>
`;
